scripting/get5.sp

```diff
@@ -360,8 +362,15 @@ public void OnPluginStart() {
   g_CheckAuthsCvar =
       CreateConVar("get5_check_auths", "1",
                    "If set to 0, get5 will not force players to the correct team based on steamid");
+  g_DateFormatCvar = CreateConVar(
+      "get5_date_format", "%Y-%m-%d",
+      "Date format to use when creating file names. Don't tweak this unless you know what you're doing! Avoid using spaces or colons.");
+  
   g_DemoNameFormatCvar = CreateConVar("get5_demo_name_format", "{MATCHID}_map{MAPNUMBER}_{MAPNAME}",
                                       "Format for demo file names, use \"\" to disable");
+  g_DemoPathCvar =
+      CreateConVar("get5_demo_path", "",
+                   "Path to save demos to, relative to the csgo directory, use \"\" to disable");
```

Comment:
Empty string does not disable, it just puts them in root, no?

Reply:
Definition updated to be in line with backup_path.